CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
   See https://github.com/mozilla/uniffi-rs/pull/1818 for details and how the other bindings were updated.
 - Removed `FfiType::RustFutureContinuationData`.  `RustFutureContinuation` callbacks now take a `u64` handle.
 
+### What's new?
+
+- Enums using proc-macros can now generate the variant literals for the foreign bindings. See
```

Comment:
I think this needs updating - I think "all" this PR is doing is exposing these variants to Swift and Kotlin?